src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
```

Comment:
You can use simple `Avenir` font-family name here and above. So in CSS you can use `Avenir` name but with a defined "font-weight".